services/account-without-juggler/repositories/account/datasources/mysql.json

```diff
@@ -0,0 +1,7 @@
+{
+    "host":"localhost",
+    "user":"root",
+    "password": "password",
+    "database": "account-service",
```

Comment:
Use the name of the example to distinguish from other databases the user may have locally before.